avocado/core/plugins/vt.py

```diff
@@ -31,6 +31,7 @@
 from avocado.core import test
 from avocado.core.settings import settings
 from avocado.core.plugins import plugin
+from avocado.utils import path
 from avocado.utils import stacktrace
```

Comment:
It's better to keep consistent with the following codes and import it as utils_path.